subprojects/gradle/build-verdict-tasks-api/src/main/kotlin/com/avito/android/build_verdict/span/SpannedString.kt

```diff
@@ -0,0 +1,29 @@
+package com.avito.android.build_verdict.span
+
+import java.net.URL
+
+data class SpannedString(
```

Comment:
This is a new type for presenting HTML parts in the build verdict by users
I tried many solutions but choose the current version. It is flexible, extensible but not a type-safe
Not type-safe means that users could set any String like an HTML and we could fail on rendering.
If we will face problems with the wrong HTML we may change API

Reply:
Maybe we should restrict clients to return only Strings for now? I mean `onTaskFailure` methods
Just convert `String` to `SpannedString` on plugin side then
Do we really need that freedom? And if yes for what particular task?
Additional benefit: backward compatibility

Reply:
I've already restricted clients from returning `String`